management-scripts/turn-off-holding-page.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+source ../cf_deployment_functions.sh
+
+check_login_variables_are_set
+
+cf_login
+
+SPACE_SUFFIX="-${CF_SPACE}"
+if [[ ${CF_SPACE} == 'production' ]]; then
+	SPACE_SUFFIX=''
+fi
+export SPACE_SUFFIX
+
+ENV_VARIABLES=`cf env help-to-buy-healthy-foods${SPACE_SUFFIX}`
+GA_TRACKING_ID=`echo "${ENV_VARIABLES}" | grep GA_TRACKING_ID | cut -d':' -f2 | cut -d',' -f1`
+UI_LOG_LEVEL=`echo "${ENV_VARIABLES}" | grep UI_LOG_LEVEL | cut -d':' -f2 | cut -d',' -f1`
+
+cf update-user-provided-service variable-service -p "'{ \"GA_TRACKING_ID\":${GA_TRACKING_ID}, \"UI_LOG_LEVEL\": ${UI_LOG_LEVEL} }'"
```

Comment:
We'll have to remember to update this whenever we add a new environment variable. At some point we may end up with a node script that extracts the current environment variables and adds/replaces the desired value - but something for the future.

Reply:
True, or a config service would be more manageable if we have lots of variables.